README.md

```diff
@@ -40,6 +40,7 @@ java -cp KafkaOffsetMonitor-assembly-0.2.1.jar \
      --port 8080 \
      --refresh 10.seconds \
      --retain 2.days
+     --brokerStorage true
```

Comment:
I needed to explicitly provide "true", as the args parsing doesn't support empty flags https://github.com/quantifind/Sumac/issues/17